mmdet/core/mask/structures.py

```diff
@@ -498,6 +520,14 @@ def crop_and_resize(self,
             resized_masks.append(resized_mask)
         return PolygonMasks(resized_masks, *out_shape)
 
+    def shear(self,
```

Comment:
We also need to implement shear operation for polygon masks.

Reply:
Hi, shear operation for polygon masks will be supported in the future version.